YAXLib/YAXLib.csproj

```diff
@@ -1,64 +1,31 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+
     <PropertyGroup>
-        <SignAssembly>true</SignAssembly>
-        <AssemblyOriginatorKeyFile>../Key/YAXLib.DevKey.snk</AssemblyOriginatorKeyFile>
-        <Version>2.16.0</Version><!-- semver -->
+        <TargetFramework>netstandard2.0</TargetFramework>
+        <PackageId>YAXLib</PackageId>
+        <Title>YAXLib</Title>
         <Description>YAXLib is an XML Serialization library which allows the programmer to structure freely the XML result, choose the fields to serialize (public, or non-public properties, or member variables), serialize all known generic and non-generic collections, serialize different kinds of arrays (single-dimensional, multi-dimensional, and jagged arrays), serialize objects through a reference to their base-class or interface (polymorphic serialization), define custom serializers, add comments for the elements in the XML result, and many more ...</Description>
-        <Copyright>Copyright (C) Sina Iravanian, Julian Verdurmen, axuno gGmbH and other contributors</Copyright>
-        <Authors>Sina Iravanian, Julian Verdurmen, axuno gGmbH and other contributors</Authors>

```

Comment:
I don't think it's that important to have my name here, but removing all the names of others without notice is a bit rude IMO

Reply:
@if @sinairv agrees, too, what about "YAXLib maintainers and other contributors"

Reply:
> I don't think it's that important to have my name here, but removing all the names of others without notice is a bit rude IMO

Uh?
This has been moved to the `Directory.Build.props` file because both `Copyright` and `Authors` should be applied to every project and not just `YAXLib.csproj`.

Reply:
Oops sorry missed that, I was unexpected we added a props file